spec/models/tag_spec.rb

```diff
@@ -96,11 +96,11 @@
 
   describe 'taggings scopes' do
     let(:first) do
-      described_class.create!(name: Faker::Science.science, creator: create(:user), status: 'approved')
+      described_class.create!(name: Faker::Science.unique.science, creator: create(:user), status: :approved)
```

Comment:
not related to PR directly, but sometimes those specs failed if Faker generated two tags with same name.